docs/source/tutorials/triton.rst

```diff
@@ -10,84 +10,57 @@ There are limits to how many of them can be supported at any point in time, shor
 so some operations typical of the Transformer family are supported in PyTorch as a sequence of base operators.
 
 Triton makes it possible to consolidate some of them into ad-hoc fused operators, which are compiled just-in-time.
-xFormers proposes a couple of optimized layers, and the goal is to increase their number over time.
+xFormers proposes a couple of optimized layers, and a goal is to increase their number  and quality over time.
 
 
 Fused softmax layer
 -------------------
 
 This is a drop-in replacement to `torch.nn.softmax`_, the only limitation being that the softmax operation is limited to the last dimension.
 Log-softmax is also available. The actual Triton kernel is very similar to `this tutorial<https://triton-lang.org/getting-started/tutorials/02-fused-softmax.html#sphx-glr-getting-started-tutorials-02-fused-softmax-py>`
+The causal option improves the performance if the attention is causal to begin with, but in that case the blocksparse attention (simply using a lower triangular layout)
```

Comment:
Might it be useful to call this out more prominently somewhere?

Reply:
it's part of what I was thinking with making this a default (once we make sure that blocksparse is bulletproof), right now I'm sure that a lot of people just walk by this optim :( maybe ok for a start and we keep updating the doc over time ?

Reply:
I'll add a line in the HOWTO